app/views/tutors/_form.html.erb

```diff
@@ -385,3 +400,41 @@
     $("#language_proficiencies_" + language).val(proficiency);
   }
 <% end %>
+
+<script>
+  (function(){
+    var tagSelect = "#tutor_all_tags";
+    var chosenSelect = "#tutor_all_tags_chosen";
```

Comment:
If you save the selector and reuse that it will improve performance. Though here it's probably not going to make a noticeable difference.